CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 3.0.0
+
+### Breaking Changes
```

Comment:
General comment for all bullet points: look for opportunities to link to the relevant section in the README that describes the new behavior at length.